frontend/appflowy_flutter/integration_test/desktop/document/document_copy_and_paste_test.dart

```diff
@@ -483,16 +483,6 @@ void main() {
       });
     });
 
-    testWidgets('paste image url without extension', (tester) async {
-      const plainText =
-          'https://images.unsplash.com/photo-1469474968028-56623f02e42e?ixlib=rb-4.0.3&q=85&fm=jpg&crop=entropy&cs=srgb&dl=david-marcu-78A265wPiO4-unsplash.jpg&w=640';
-      await tester.pasteContent(plainText: plainText, (editorState) {
-        final node = editorState.getNodeAtPath([0])!;
-        expect(node.type, ImageBlockKeys.type);
-        expect(node.attributes[ImageBlockKeys.url], isNotEmpty);
-      });
-    });
-
```

Comment:
why do we need to remove this test?

Reply:
We now not support paste an image-url as an image due to the `Paste as` menu, because to check whether a url is an image via a http request will cost a lot of time, it will block the UI